docs/tutorial/node-app.rst

```diff
@@ -116,9 +116,20 @@ service:
     2023-10-30T12:37:33.663Z [pebble] Service "app" starting: node server.js
     2023-10-30T12:37:33.864Z [app] Running on http://0.0.0.0:8080
 
```

Comment:
https://canonical-ubuntu-documentation-library--652.com.readthedocs.build/rockcraft/en/652/tutorial/node-app/#run-the-rock-in-docker